lib/fluent/plugin/out_vmware_loginsight.rb

```diff
@@ -70,7 +70,16 @@ class ConnectionFailure < StandardError; end
       config_param :flatten_hashes, :bool, :default => true
       # Seperator to use for joining flattened keys
       config_param :flatten_hashes_separator, :string, :default => "_"
-
+      # Keys from log event whose values shorten
+      config_param :shorten_keys, :hash, default:
```

Comment:
Can you please add `value_type: :string`?

Reply:
done